identity/utils.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+from django.conf import settings
+from jwcrypto import jwk, jwt
+
+
+SIGNING_ALG = "RS256"
+EXPIRY_TIME = 15 * 60  # 15 minutes
+id_privkey = jwk.JWK.from_pem(settings.IDENTITY_RSA_PRIVATE_KEY.encode("utf-8"))
```

Comment:
Super nit, but I think this should be in all caps since it's technically a constant, right?

Reply:
yeah